data-mermaid/mermaid-webapp
# M645 ensure project list info is up-to-date (by refreshing the current user profile which stores some project info)

[Ticket](https://trello.com/c/IHU659Y1/645-need-to-log-out-to-refresh-the-project-page)

Steps to test (from ticket):

> go to Projects page, note number of unsubmitted transects owned by me (in my case, 4)
> enter a CR (just enough to be able to save; I just chose a site) and save; observer red counter by your name increment (in my case, 5)
> go back to Projects page, navigate to relevant project, observe red counter (in my case, 4)
